src/test/java/hw1/CalculatorBaseTest.java

```diff
@@ -0,0 +1,22 @@
+package hw1;
+
+import com.epam.tat.module4.Calculator;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+
+public class CalculatorBaseTest {
+
+    protected Calculator calculator;
+    protected static final double DOUBLE_DELTA = 1e-6;
+
+    @BeforeClass
```

Comment:
why did you decide to use @BeforeClass, @AfterClass?

Reply:
I think that it's a bit redundant to init new Calculator before each test. So I tried to decide what's better @BeforeClass or @BeforeSuite. But in this tests there are no difference beetwen annotations. Now changed to @BeforeSuite

Reply:
Because I think that @BeforeTest is a bit redundant